src/os/KeyValueDB.cc

```diff
@@ -11,21 +11,23 @@
 #endif
 
 KeyValueDB *KeyValueDB::create(CephContext *cct, const string& type,
-			       const string& dir)
+			       const string& dir, const bool default_sync_transaction)
```

Comment:
The one I'm not sure about is KeyValueStore... I'm not sure how it is managing the commits.